src/main/java/cloud/fogbow/ras/core/OrderController.java

```diff
@@ -410,6 +466,22 @@ ComputeAllocation getUserComputeAllocation(Collection<ComputeOrder> computeOrder
 
         return requestedOrders;
     }
+    
+    // FIXME should also receive provider id
+    private List<Order> getOrdersByUserId(String userId, String providerId, ResourceType resourceType) {
```

Comment:
It seems this comment is outdated. Remove it. Otherwise, do the changes needed.

Reply:
Done